scio-core/src/main/scala/com/spotify/scio/values/SideInput.scala

```diff
@@ -127,6 +128,7 @@ private[values] class MultiMapSideInput[K, V](val view: PCollectionView[JMap[K,
     JMapWrapper.ofMultiMap(context.sideInput(view))
 }
 
+@deprecated(since = "0.14.0")
```

Comment:
can we include a deprecation message advising users what methods to use instead?

Reply:
No this is just a pure "don't do this"